recipes/LibriSpeech/ASR/transducer/hparams/conformer_transducer.yaml

```diff
@@ -213,16 +219,17 @@ dec: !new:speechbrain.nnet.RNN.LSTM
    hidden_size: !ref <dec_dim>
    num_layers: 1
    re_init: True
+   dropout: 0.1
 
 # For MTL with LM over the decoder
 dec_lin: !new:speechbrain.nnet.linear.Linear
-   input_size: !ref <dec_dim>
-   n_neurons: !ref <joint_dim>
+   input_size: !ref <joint_dim>
+   n_neurons: !ref <output_neurons>
    bias: False
 
 # For MTL
-#ce_cost: !name:speechbrain.nnet.losses.nll_loss
-#   label_smoothing: 0.1
+ce_cost: !name:speechbrain.nnet.losses.nll_loss
```

Comment:
Note that `ce_weight` is `0.0` so this isn't effective right now (stating just in case).

Is it possible that applying label smoothing over the auxiliary loss here would help the other losses? I would think not but I am wondering.